charts/fluent-operator/values.yaml

```diff
@@ -0,0 +1,182 @@
+# Default values for fluentbit-operator.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+# Set this to containerd or crio if you want to collect CRI format logs
+containerRuntime: docker
+# You can also deploy the Kubernetes logging stack with Helm, just need to set the Kubernetes parameter to ture.
+# see https://github.com/fluent/fluent-operator/tree/master/manifests/logging-stack
+Kubernetes: false
+
+operator:
+# The initial container is to get the actual storage path of the docker log files so that it can be mounted to collect the logs.
+# see https://github.com/fluent/fluent-operator/blob/master/manifests/setup/fluent-operator-deployment.yaml#L26
+  initcontainer:
+    repository: "docker"
+    tag: "20.10"
+  container:
+    repository: "kubesphere/fluent-operator"
+    tag: "v1.7.0"
+    # FluentBit operator resources. Usually user needn't to adjust these.
+  resources:
+    limits:
+      cpu: 100m
+      memory: 60Mi
+    requests:
+      cpu: 100m
+      memory: 20Mi
+  ## Reference to one or more secrets to be used when pulling images
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/pull-image-private-registry/
+  imagePullSecrets: []
+  # - name: "image-pull-secret"
+  # For non-docker environments, the mount path for fluentbit uses a hard-coded path.
+  logPath:
+    crio: /var/log
+    containerd: /var/log
+
+fluentbit:
+  image:
+    repository: "kubesphere/fluent-bit"
+    tag: "v1.9.9"
+  # fluentbit resources. If you do want to specify resources, adjust them as necessary
+  # You can adjust it based on the log volume.
+  resources:
+    limits:
+      cpu: 500m
+      memory: 200Mi
+    requests:
+      cpu: 10m
+      memory: 25Mi
+  # Specify custom annotations to be added to each FluentBit pod.
+  annotations: {}
```

Comment:
I think it would be good to use the exclude annotation here by default to be honest for two reasons:
- it is a great working example people can replicate for other pods
- it prevents a recursive log snowball effect (e.g. FB logs an error saying it cannot send to an output, FB then reads its own log and tries to send this new error message to the output which then triggers another error...)

It also means we can use `stdout` by default or for debugging easily for new users.

Reply:
https://docs.fluentbit.io/manual/pipeline/filters/kubernetes#kubernetes-annotations

Reply:
@patrick-stephens collecting fluentbit container logs is also valuable to debug fluentbit's issue in case of misconfiguration or other cases. Simply excluding fluentbit's own logs might not work for this kind of requirements

Reply:
Sure, let's have a commented out example then for people.

Reply:
> Sure, let's have a commented out example then for people.
Agree, @wenchajun we need to comment out the settings to exclude FB logs and the setting of output to stdout